src/locales/en.json

```diff
@@ -166,6 +172,7 @@
     "risk_group_empty_warning": "Enter your location first.",
     "saved_scenario_loaded_message": "If needed, adjust the following prefilled answers to match your <strong>{{ scenarioName }}</strong> scenario.",
     "alerts": {
+      "vaccines_update": "<0> February 23rd, 2021:</0> We added a precaution section for being vaccinated. Learn more in the <2>Q&A</2>.",
```

Comment:
I think this is too jargon-y. If it was my first time visiting the calculator, I would have no reason to know what a precaution section is. Thoughts on:

* We added a "your vaccine" precaution.
* The calculator now gives a bonus if you are vaccinated.

Reply:
Done.

Reply:
Done.